circle.yml

```diff
@@ -24,6 +24,7 @@ dependencies:
 
 test:
   override:
+    - cd "$PROJECT_PATH" && make get_vendor_deps && make metalinter_test
```

Comment:
can we do the metalinter test without vendor deps? we're going to download the vendor deps later ... no need to do it twice